build.gradle

```diff
@@ -47,6 +47,15 @@ idea.project {
   languageLevel = JavaVersion.VERSION_1_7
 }
 
+// Gobblin build customization
+if (!project.hasProperty('gobblinFlavor')) {
```

Comment:
Probably makes sense to add this in build documentation and integration docs for kafka etc. 
Will open separate PR for that later.